packages/medusa/src/api/routes/admin/collections/index.ts

```diff
@@ -0,0 +1,40 @@
+import { Router } from "express"
+import { ProductCollection } from "../../../.."
+import { DeleteResponse, PaginatedResponse } from "../../../../types/common"
+import middlewares from "../../../middlewares"
+import "reflect-metadata"
```

Comment:
Is this used?

Reply:
yeah, everytime we use `@Type` this should be imported